lib/pay_day_loan.ex

```diff
@@ -182,11 +279,21 @@ defmodule PayDayLoan do
 
   @doc """
   Check for cached pid, but do not request a load
+
+  This is a legacy API method and may be deprecated.  Use `peek/2`.
   """
   @spec peek_pid(pdl :: t, key :: PayDayLoan.key)
   :: {:ok, pid} | {:error, :not_found}
   def peek_pid(pdl = %PayDayLoan{}, key) do
-    CacheStateManager.get_pid(pdl.cache_state_manager, key)
+    peek(pdl, key)
```

Comment:
I'm tempted just to remove the `_pid` functions completely.  I don't know who else if anyone is using this and being < 1.0 it's fair game.  I'd try to communicate this somehow in the README and/or changelog and/or release notes.